tox.ini

```diff
@@ -1,13 +1,23 @@
 # This file is part of REANA.
-# Copyright (C) 2022, 2023 CERN.
+# Copyright (C) 2022, 2023, 2024 CERN.
 #
 # REANA is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
 
 [tox]
-envlist = py38, py39, py310, py311, py312
+envlist =
+    py38
+    py39
+    py310
+    py311
+    py312
```

Comment:
Note: for Python 3.12, to run `tox -e py312` locally, we need to first release new `pytest-reana` that is bumping the limit of the `coverage` dependency. This is done in https://github.com/reanahub/pytest-reana/pull/134

Reply:
```
$ sudo dnf install python3.8 python3.9 python3.10 python3.11 python3.12
$ sudo dnf install python3.10-devel python3.11-devel python3.12-devel
$ tox --skip-env py312
...
  py38: OK (43.40=setup[14.36]+cmd[11.09,17.95] seconds)
  py39: OK (55.55=setup[21.52]+cmd[16.22,17.80] seconds)
  py310: OK (54.54=setup[20.63]+cmd[15.61,18.31] seconds)
  py311: OK (73.71=setup[1.25]+cmd[5.53,66.94] seconds)
  congratulations :) (227.31 seconds)
```